backend/src/main/java/com/redhat/cloud/notifications/routers/sources/SourcesService.java

```diff
@@ -0,0 +1,84 @@
+package com.redhat.cloud.notifications.routers.sources;
+
+import io.quarkus.rest.client.reactive.ClientExceptionMapper;
+import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PATCH;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Response;
+
+/**
+ * <p>REST Client for the Sources API. The OpenAPI spec is available at:</p>
+ *
+ * <ul>
+ *  <li><a href="https://console.stage.redhat.com/api/sources/v3.1/openapi.json">OpenApi v3.1 in the stage environment.</a></li>
+ *  <li><a href="https://console.redhat.com/api/sources/v3.1/openapi.json">OpenApi v3.1 in the production environment.</a></li>
```

Comment:
Links to https://console.redhat.com/docs/api/sources/v3.1 or the equivalent for stage could be helpful here. We can still find the raw JSON files from there.

Reply:
I think I included links to the stage, production and then the GitHub repository directly. Maybe I missed something?

Reply:
Sorry, my comment wasn't clear enough. I meant that a link to https://console.redhat.com/docs/api/sources/v3.1 is more helpful (from my perspective) than a link to https://console.redhat.com/api/sources/v3.1/openapi.json because of the user-friendly UI 😃

Reply:
Oooooh sure, my bad! Yeah, I agree the Swagger UI is just better. Changing it!